### PR TITLE
PUBDEV-8751: Added `transform_frame` to GLRM

### PR DESCRIPTION
**Pull request by hannah-tillman**
For: [PUBDEV-8751](https://h2oai.atlassian.net/browse/PUBDEV-8751)

I wasn't sure if "transforming data" was a correct header, so please let me know if that or something else needs to be updated :)

### CODE REVIEW

File: h2o-docs/src/product/data-science/glrm.rst [hunk 1]
```diff
@@ -125,6 +152,10 @@ Below is a simple example showing how to build a Generalized Low Rank model.
     # Generate predictions on a validation set (if necessary):
     arrests_pred <- h2o.predict(glrm_model, newdata = valid)
 
+    # Transform the data using the validation set to retrieve the new coefficients:
+    glrm_transform <- h2o.transform_frame(glrm_model, valid)
```
Comment (wendycwong) [inline h2o-docs/src/product/data-science/glrm.rst:156]: @hannah-tillman Can you change the wording:

using the validation set

to 

using the dataset valid

File: h2o-docs/src/product/data-science/glrm.rst [hunk 2]
```diff
@@ -149,6 +180,9 @@ Below is a simple example showing how to build a Generalized Low Rank model.
                                                 transform="standardize")
     glrm_model.train(training_frame=train) 
 
+    # Transform the data using the validation set to retrieve the new coefficients:
+    glrm_transform = glrm_model.transform_frame(valid)
+
```
Comment (wendycwong) [inline h2o-docs/src/product/data-science/glrm.rst:185]: @hannah-tillman Can you change the wording:

using the validation set

to

using the dataset valid

**Review by wendycwong — CHANGES_REQUESTED** [review]

**Review by wendycwong — APPROVED** [review]
LGTM.  Thanks!
